docs/docs/hpc-on-k8s/user.md

```diff
@@ -0,0 +1,129 @@
+# Steps to create bio-user in bio namespace
+
```

Comment:
I think this file needs some explanation of:
- What the users are for
- What the users have access to

Reply:
updated thanks